QuizApi/Controllers/TestController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using QuizApp.DTO;
+using QuizApp.Services.Contracts;
+
+namespace QuizApp.Controllers
+{
+	[Route("api/[controller]")]
+	[ApiController]
+	public class TestController : ControllerBase
+	{
+		private readonly ITestService _testService;
+
+		public TestController(ITestService testService)
+		{
+			_testService = testService;
```

Comment:
Please add a check for null reference

Reply:
Added check 
_testService ??= testService;
Or there should be testService check and Exception if its null?
_testService = testService ?? throw new ArgumentNullException(nameof(testService));

Reply:
`_testService = testService ?? throw new ArgumentNullException(nameof(testService));`
this one would be perfect
But let's do this when we will have global exception handling.
You should check the argument for null